CHANGELOG.md

```diff
@@ -1,7 +1,80 @@
 # TimescaleDB Changelog
 
-**A revised version of 0.9.0 is forthcoming soon. Please do not attempt
-to update yet**
+## 0.9.0 (2018-03-05)
+
+**High-level changes**
+* Support for multiple extension versions on different databases in the
+same PostgreSQL instance. This allows different databases to be updated
+independently and provides for smoother updates between versions. No
+more spurious errors in the log as the extension is being
+updated, and new versions no longer require a restart of the database.
+* Streamlined update process for smaller binary/package sizes
+* Significant refactoring to simplify and improve codebase, including
+improvements to error handling, security/permissions, and more
+* Corrections to edge-case scenarios involving dropping schemas,
+hypertables, dimensions, and more
+* Correctness improvements through propagating reloptions from main
+table to chunk tables and blocking `ONLY` commands that try to alter
+hypertables (i.e., changes should be applied to chunks as well)
+* Addition of a `migrate_data` option to `create_hypertable` to allow
+non-empty tables to be turned into hypertables without separate
+creation & insertion steps. Note, this option may take a while if the
+original table has lots of data
+* Support for `ALTER TABLE RENAME CONSTRAINT`
+* Support for adjusting the number of partitions for a space dimension
+* Improvements to tablespace handling
+
+**Notable commits**
+* [4672719] Fix error in handling of RESET ALL
+* [9399308] Refactor/simplify update scripts and build process
+* [7bd9437] Do not generate versioned copies of SQL files
```

Comment:
A lot of these commit don't seem very notable... not a big deal though

Reply:
Oops I think I left some I meant to remove (I start by copying over all of them and then manually removing).